src/lib/utils.ts

```diff
@@ -45,6 +45,9 @@ export const formatTimestamp = (timestamp: bigint, withDate = true) => {
 };
 
 
+export function formatAddress(address: string) {
+  return `${address.slice(0, 10)}...${address.slice(-8)}`;
+
```

Comment:
You're missing a closing curly brace here, this makes the whole project not starting.

Reply:
fixed also